.github/dependabot.yml

```diff
@@ -0,0 +1,24 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
```

Comment:
Out of scope for this PR: One thought that I had here is whether it makes sense to add a more regular check for `github.com/NVIDIA` dependencies here?